buildpack/nginx.py

```diff
@@ -236,6 +238,18 @@ def get_path_config(samesite_cookie_workaround=False):
         if config.get("client-cert") or config.get("client_cert"):
             client_cert = "auth_request /client-cert-check-internal;"
 
+        issuer_dn = ""
```

Comment:
Will this break if we initialise this as `None`? Python is quite different than Java in handling "empty" values.

Reply:
I would leave this for now.